packages/jest-mocks/index.ts

```diff
@@ -33,7 +33,10 @@ export function mockConsole(
   }
 
   if (result && typeof result.then === "function") {
-    return result.then(restoreConsole).catch(restoreConsole)
+    return result.then(restoreConsole).catch((e) => {
+      restoreConsole()
+      throw e
+    })
```

Comment:
mockConsole function muted all caught errors. That makes tests pass, when it's actually shouldn't